R/template.R

```diff
@@ -233,16 +240,20 @@ prob_metric_summarizer <- function(name,
 
   out <- dplyr::summarise(
     data,
-    .metric = name,
-    .estimator = finalize_estimator(.data[[truth]], estimator, name),
+    .metric = .env[["name"]],
+    .estimator = finalize_estimator(
+      .data[[truth]],
+      .env[["estimator"]],
+      .env[["name"]]
+    ),
     .estimate = fn(
       truth = .data[[truth]],
       estimate = {
         estimate <- dplyr::pick(tidyselect::all_of(estimate))
```

Comment:
```suggestion
        estimate <- dplyr::pick(tidyselect::all_of(.env[["estimate"]]))
```

Check me on this one, maybe add a test for it